frontend/sgp/src/app/pages/questao/form/questao.component.ts

```diff
@@ -87,7 +87,7 @@ export class QuestaoComponent implements OnInit {
   }
 
   salvar() {
-    if (this.questao.id === null) {
+    if (typeof this.questao.id === 'undefined' || this.questao.id == null) {
```

Comment:
essa verificação pode ser feita assim: if(!Boolean(this.questao.id))

Reply:
Vou mudar